stabilizer/src/main/java/com/hazelcast/stabilizer/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,239 @@
+package com.hazelcast.stabilizer.utils;
+
+import com.hazelcast.stabilizer.test.annotations.Name;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.LinkedList;
+import java.util.List;
+
+import static java.lang.String.format;
+
+public class ReflectionUtils {
```

Comment:
Can you also add some tests for this class? Don't need to go into all potential cases, but just a few smoke tests to see if it works.